web/react/stores/browser_store.jsx

```diff
@@ -1,85 +1,104 @@
 // Copyright (c) 2015 Spinpunch, Inc. All Rights Reserved.
 // See License.txt for license information.
-var UserStore = require('../stores/user_store.jsx');
 
-// Also change model/utils.go ETAG_ROOT_VERSION
-var BROWSER_STORE_VERSION = '.1';
-
-var _initialized = false;
-
-function _initialize() {
-	var currentVersion = localStorage.getItem("local_storage_version");
-	if (currentVersion !== BROWSER_STORE_VERSION) {
-		localStorage.clear();
-		sessionStorage.clear();
-		localStorage.setItem("local_storage_version", BROWSER_STORE_VERSION);
-	}
-    _initialized = true;
+var UserStore;
+function getPrefix() {
+    if (!UserStore) UserStore = require('./user_store.jsx');
+    return UserStore.getCurrentId() + '_';
 }
 
-module.exports.setItem = function(name, value) {
-    if (!_initialized) _initialize();
-	var user_id = UserStore.getCurrentId();
-	localStorage.setItem(user_id + "_" + name, value);
-};
+// Also change model/utils.go ETAG_ROOT_VERSION
+var BROWSER_STORE_VERSION = '.3';
 
-module.exports.getItem = function(name) {
-    if (!_initialized) _initialize();
-	var user_id = UserStore.getCurrentId();
-	return localStorage.getItem(user_id + "_" + name);
-};
+module.exports = {
+    _initialized: false,
 
-module.exports.removeItem = function(name) {
-    if (!_initialized) _initialize();
-	var user_id = UserStore.getCurrentId();
-	localStorage.removeItem(user_id + "_" + name);
-};
+    _initialize: function() {
+        var currentVersion = localStorage.getItem("local_storage_version");
+        if (currentVersion !== BROWSER_STORE_VERSION) {
+            this.clear();
+            localStorage.setItem("local_storage_version", BROWSER_STORE_VERSION);
+        }
+        this._initialized = true;
+    },
 
-module.exports.setGlobalItem = function(name, value) {
-    if (!_initialized) _initialize();
-	localStorage.setItem(name, value);
-};
+    getItem: function(name, defaultValue) {
```

Comment:
getItem and setItem should work the same way as their corresponding global functions. So they should stringify and parse.